Makefile

```diff
@@ -43,6 +42,8 @@ ifeq ($(OS),linux)
 
 LDLIBS=\
 	-lrt\
+	-ljemalloc
+	-lsystemd
```

Comment:
Not all users/systems has systemd/jemalloc so this must be optional and disabled by default.

And there is a typo:
```diff
LDLIBS=\
	-lrt\
-	-ljemalloc
-	-lsystemd
+	-ljemalloc\
+	-lsystemd\
```